kernel/aster-nix/src/device/mod.rs

```diff
@@ -3,17 +3,17 @@
 mod null;
 mod pty;
 mod random;
-#[cfg(feature = "intel_tdx")]
+#[cfg(all(target_arch = "x86_64", feature = "intel_tdx"))]
```

Comment:
Since we are adding RISC-V support and Cargo does not support CPU architecture-specific features, it seems that we should make all Cargo features CPU architecture _neutral_. For example, instead of having a feature called `intel_tdx`, we should have the feature named `cvm_guest` (the guest OS of a Confidential VM). What do you think?